package.json

```diff
@@ -22,6 +22,7 @@
     "@blockstack/keychain": "^0.1.0",
     "@blockstack/prettier-config": "^0.0.3",
     "@blockstack/ui": "^1.0.0-alpha.6",
+    "@schemastore/web-manifest": "^0.0.4",
```

Comment:
I think this should be in `devDependencies` - no? Since it's just a Typescript type we're using, I think we don't need it in the main package. Maybe I'm wrong.

Reply:
You're right, missed `-D` flag